DBTEMPLATE/post-refresh/scripts/dxh_wrapper_script.sh

```diff
@@ -47,6 +47,8 @@ HOOK_OP=${4}
 # sample code to optimize a VDB for Masking (expects ORACLE_SID and ORACLE_HOME to be set)
 # ${BASEDIR}/ora_vdb_masking_prep.sh 
 
+#. $BASEDIR/recreate_users_MT.sh $DELPHIX_PDB_NAME
```

Comment:
I don't think we want a  "." here.  If it's an environment file, we use "." or `source`.  

Technically, I think your command will work fine, it's just confusing style (to me at least).  Unless we are sure the "." is needed (in other words, you want to run this from the current shell) we should take it out.

https://superuser.com/questions/176783/what-is-the-difference-between-executing-a-bash-script-vs-sourcing-it#:~:text=7%20Answers&text=Sourcing%20a%20script%20will%20run,in%20your%20currently%20running%20shell.